src/client/contextmenu.js

```diff
@@ -68,11 +68,14 @@ export default class ContextMenu {
       }, "CMD+K"],
       ["Browser",     (evt) => {
         this.openComponentInWindow("lively-container", evt).then(comp => {
-          var container = _.last(document.querySelectorAll("lively-container"));
```

Comment:
This would match *all* `lively-container`s that are the last element in their parents. Given the example below both would be matched:

```
<div><lively-container /></div>
<div><lively-container /></div>
```